code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -197,63 +228,77 @@
 //nuke op pinpointers//
 ///////////////////////
 /obj/item/pinpointer/nukeop
-	var/mode = 0	//Mode 0 locates disk, mode 1 locates the shuttle
 	var/obj/docking_port/mobile/home = null
 	slot_flags = SLOT_BELT | SLOT_PDA
+	syndicate = TRUE
 
 /obj/item/pinpointer/nukeop/attack_self(mob/user as mob)
-	if(!active)
+	if(active == FALSE && !mode)
 		active = 1
-		if(!mode)
-			workdisk()
-			to_chat(user, "<span class='notice'>Authentication Disk Locator active.</span>")
-		else
-			worklocation()
-			to_chat(user, "<span class='notice'>Shuttle Locator active.</span>")
+		workdisk()
+		to_chat(user, "<span class='notice'>Authentication Disk Locator active.</span>")
+	else if(active == 1 && !mode)
+		active = 2
+		workbomb()
+		to_chat(user, "<span class='notice'>Nuclear Device Locator active.</span>")
+	else if(mode && !active == 1)
```

Comment:
`!active == 1`? This only works if `active == 0`

Reply:
Changing to just !active, as we want this to turn on when shuttle locator mode (mode) is true, and we want it to only switch to this mode if it is not active